pom.xml

```diff
@@ -98,6 +98,8 @@
 
     <!-- FIXME: Overrides OSS parent pom for java 16 compatibility -->
     <version.artifactory.plugin>3.2.1</version.artifactory.plugin>
+
+    <sonar.skip>true</sonar.skip>
```

Comment:
Is this a temporary change? If so, it might be worth leaving a meaningful comment next to it.

Reply:
Yes, but I think it's fine as it is, I will directly create a PR to not forget it.